docs/cli_options/client_options.rst

```diff
@@ -382,6 +382,16 @@ enforced only when spawning at least one worker process:
         def test(self):
             pass
 
+.. warning::
+
+    When running with multiprocess you can't use IPDBugger (--debug).
+
+.. warning::
+
+    It is **not** recommended using this option when you have a SQLite database,
```

Comment:
Is it possible to disallow it from happening?
Like raising an exception if the two are combined

Reply:
It is possible, but complicated
since some tests can use only services and don't need resources, so they won't get an error